contribs/gmf/src/directives/mobilemeasurelength.js

```diff
@@ -0,0 +1,216 @@
+goog.provide('gmf.MobileMeasureLengthController');
+goog.provide('gmf.mobileMeasureLengthDirective');
+
+goog.require('gmf');
+goog.require('ngeo.interaction.MobileDraw');
+
+gmfModule.value('gmfMobileMeasureLengthTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfMobileMeasureLengthTemplateurl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/mobilemeasurelength.html';
+    });
+
+
+/**
+ * Provide a "mobile measure" directive.
+ *
+ * @example
+ * <div gmf-mobile-measure-length=""
+ *   gmf-mobile-measure-length-map="ctrl.map"
+ *   gmf-mobile-measure-length-draw-line="ctrl.mobileDrawLine">
+ * </div>
+ *
+ * @param {string|function(!angular.JQLite=, !angular.Attributes=)}
+ *     gmfMobileMeasureLengthTemplateUrl Template URL for the directive.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfMobileMeasureLength
+ */
+gmf.mobileMeasureLengthDirective =
+    function(gmfMobileMeasureLengthTemplateUrl) {
+  return {
+    restrict: 'A',
+    scope: {
+      'map': '=gmfMobileMeasureLengthMap',
+      'mobileDrawLine': '=gmfMobileMeasureLengthDrawLine'
+    },
+    controller: 'GmfMobileMeasureLengthController',
+    controllerAs: 'measureLengthCtrl',
```

Comment:
i think it's a bit too long for nothing, when in a directive with isolate scope, you can use `ctrl`, or `mlCtrl`.
It's not dramatic but it's maybe better =)